docs/helpers/Ticks.md

```diff
@@ -107,3 +108,21 @@ var axis = {
     }
 };
 ```
+
+## Tick Values
+
+The tick values are set one of 3 methods, in the following priority:
+
+-   **Consumer provided values**:
+
+    Consumer provided values in `axis.y.ticks.values` and `axis.y2.ticks.values` will take priority if provided.
+
+-   **ticksCount property**
+
+    If no values are provided and the ticksCount property is set, then it will be used to calculate tick values by dividing `upperLimit - lowerLimit` into `ticksCount + 1` equal divisions. If ticksCount is an invalid value, such as greater than `TICKSCOUNT_MAXLIMIT`, then this property will be ignored and the default behavior will be used.
```

Comment:
As long as `config.axis.y.ticks.values` is defined, custom ticks will take a priority.

Reply:
Can we add a note about this? If consumer provides both `custom values` and `ticksCount`, custom values takes priority?

So that consumer will know about this behavior